src/constants/routes.constants.coffee

```diff
@@ -4,7 +4,7 @@ routes_Names =
   components: {}
   views:
     # defined via dedicated state: '/user/article'
-    guest     : ['about',  'features', 'home', 'login', 'pwd_forgot', 'pwd_reset', 'sign_up']
+    guest     : ['about',  'features', 'home', 'login', 'pwd_forgot', 'sign_up']
```

Comment:
humm, why are you removing this view, I'm going to merge and double check, but I believe you will need it